### ISSUE TITLE
ml-track

### ISSUE THREAD

**Issue by ntwong0** [SJSU-Robotic/ml-curriculum#13]

**Comment by ntwong0** [comment]
@josh-github1 Good meeting - please 

1. fork this repo (if you haven't already),
2. upload your solution to `svm.ipynb`, and
3. link it here

See you next week

**Comment by josh-github1** [comment]
still working on softmax assignment but I also had trouble understanding regularization and the numpy implementation of backpropagation for 2 layer NN's
